frontend/appflowy_flutter/lib/plugins/document/presentation/editor_plugins/base/insert_page_command.dart

```diff
@@ -25,23 +25,23 @@ extension InsertDatabase on EditorState {
       return;
     }
 
-    // get the database that the view is associated with
-    final database = await DatabaseViewBackendService(viewId: childView.id)
-        .openGrid()
+    // get the database id that the view is associated with
+    final databaseId = await DatabaseViewBackendService(viewId: childView.id)
+        .getDatabaseId()
         .then((value) => value.swap().toOption().toNullable());
 
-    if (database == null) {
+    if (databaseId == null) {
       throw StateError(
         'The database associated with ${childView.id} could not be found while attempting to create a referenced ${childView.layout.name}.',
       );
     }
 
     final prefix = _referencedDatabasePrefix(childView.layout);
     final ref = await ViewBackendService.createDatabaseReferenceView(
-      parentViewId: parentView.id,
+      parentViewId: childView.id,
```

Comment:
if you assign the `childView.id` to this parameter, we should consider to rename it's name.